libs/api/domains/auth/src/lib/models/delegation.model.ts

```diff
@@ -68,6 +62,12 @@ export class CustomDelegation extends Delegation {
   @Field((type) => ID)
   id!: string
 
+  @Field((type) => Date, { nullable: true })
+  validTo?: Date
+
+  @Field((type) => [String])
+  scopesNames!: string[]
```

Comment:
scopeNames?

Reply:
Do we need this?

Reply:
Yeah, I can delete this! The original model typed the scope as string array, but I can see the field isn't queried, so I'll delete it 👍